store/store.go

```diff
@@ -1255,9 +1257,69 @@ func (s *Store) downloadDelta(name string, downloadDir string, downloadInfo *sna
 		return "", err
 	}
 
+	// TODO: Check sha3_384
 	return deltaPath, nil
 }
 
+// applyDelta generates a target snap from the current snap and a downloaded delta.
+var applyDelta = func(name string, deltaPath string, deltaInfo *snap.DeltaInfo) (string, error) {
+	currentSnapName := fmt.Sprintf("%s_%d.snap", name, deltaInfo.FromRevision)
```

Comment:
Done - well, s/currentSnapName/snapBase and s/currentSnapPath/snapPath/ as per chat.